types/pxr-oneline/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "es7", "dom"
+        ],
```

```suggestion
            "es6",
            "dom"
        ],
```
format. It works with ES6, what's behind ES7 here?